EKF/estimator_interface.cpp

```diff
@@ -314,7 +314,7 @@ void EstimatorInterface::setAirspeedData(uint64_t time_usec, float true_airspeed
 	}
 }
 
-void EstimatorInterface::setRangeData(uint64_t time_usec, float data)
+void EstimatorInterface::setRangeData(uint64_t time_usec, float data, uint8_t quality)
```

Comment:
It is preferable to use the same data type and definitions for quality as the distance_sensor topic.